pallets/dapps-staking/src/pallet/mod.rs

```diff
@@ -332,9 +348,9 @@ pub mod pallet {
 
                 Self::deposit_event(Event::<T>::NewDappStakingEra(next_era));
 
-                consumed_weight + T::DbWeight::get().reads_writes(5, 3)
+                consumed_weight + T::DbWeight::get().reads_writes(8, 3)
             } else {
-                T::DbWeight::get().reads(4)
+                T::DbWeight::get().reads(7)
```

Comment:
Why we need to bump 3 reads here when only one new read (`DecomissionStarted `) is introduced ? I have trouble understanding this

Reply:
Same reason as above.